tribe-ext-relabeler.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/**
+ * Plugin Name:         The Events Calendar Extension: Relabeler
+ * Description:         Adds option to WP Admin > Events > Display for altering labels. For example, you can change the word "Events" to a different word such as "Gigs".
+ * Plugin URI:          https://theeventscalendar.com/extensions/change-labels-events-venues-organizers/
+ * GitHub Plugin URI:   https://github.com/mt-support/tribe-ext-relabeler
+ * Version:             1.1.0
+ * Extension Class:     Tribe\Extensions\Relabeler\Main
+ * Author:              Modern Tribe, Inc.
+ * Author URI:          http://m.tri.be/1971
+ * License:             GPLv3 or later
+ * License URI:         https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:         tribe-ext-relabeler
+ */
+
+namespace Tribe\Extensions\Relabeler;
+
+use Tribe__Autoloader;
+use Tribe__Extension;
+
+/**
+ * Define Constants
+ */
+
+if ( ! defined( __NAMESPACE__ . '\NS' ) ) {
+	define( __NAMESPACE__ . '\NS', __NAMESPACE__ . '\\' );
+}
+
+if ( ! defined( \Tribe\Extensions\Relabeler\NS . 'PLUGIN_TEXT_DOMAIN' ) ) {
+	// `Tribe\Extensions\Example\PLUGIN_TEXT_DOMAIN` is defined
+	define( NS . 'PLUGIN_TEXT_DOMAIN', 'tribe-ext-relabeler' );
+}
+
```

Comment:
consider removing and replacing with plain text domain usage throughout, per latest extension template